documentation.md

```diff
@@ -0,0 +1,245 @@
+
+#### Security announcements
+
+| Attribute                 | Type          | Description                       | Exemple of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| type                      | String        | Security type                     | "SecurityAnnouncements::Ubuntu"                   |
+| sa_code                   | String        | Security alert code               | "USN-3938-1"                                      |
+| created_at                | String (date) | Date of created                   | "2019-04-08T02:00:00.000+02:00"                   |
+| updated_at                | String (date) | Date of updated                   | "2019-04-10T09:20:07.000+02:00"                   |
+| link                      | String        | Link of the security              | "https://usn.ubuntu.com/3938-1"                   |
+| cve_code                  | String        | CVE Code public                   | "CVE-2019-3842"                                   |
+
+#### Updates
+
+| Attribute                 | Type          | Description                       | Exemple of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| ignored                   | Bool          | Ignore of update                  | false                                             |
+| patchable                 | Bool          | Update is possible                | true                                              |
+| cve_announcements         | Object        | All the update availbale cve      | [CveAnnouncements](#Cve-announcements)            |
+| current                   | Object        | Current version of packages       | [Current](#Current)                               |
+| target                    | Object        | Target version of packages        | [Target](#Target)                                 |
+| created_at                | String (date) | Date of created                   | "2019-04-09T17:13:07.000+02:00"                   |
+| updated_at                | String (date) | Date of updated                   | "2019-04-09T17:13:07.000+02:00"                   |
+
+#### Cve announcements
+
+| Attribute                 | Type          | Description                       | Exemple of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| id_cve                    | Int           | Id Cyberwatch of the vulnerability| 22                                                |
+| server_id                 | Int           | Id Cyberwatch of the server       | 21                                                |
```

Comment:
does the `cve_announcements` object even has an attribute `server_id` ?

I'm getting : 
```python
>>> print(serv.cve_announcements[0].server_id)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'CBWCve' object has no attribute 'server_id'
```

and the code of the class cbw_cve doesn't seem to have one either :
```python
"""CVE Model"""


class CBWCve:
    """CVE Model"""

    def __init__(self,
                 content="",
                 created_at="",
                 cve_code="",
                 cve_score="",
                 last_modified="",
                 published="",
                 updated_at="",
                 **kwargs): # pylint: disable=unused-argument
        self.content = content
        self.created_at = created_at
        self.cve_code = cve_code
        self.cve_score = cve_score
        self.last_modified = last_modified
        self.published = published
        self.updated_at = updated_at
```